source/runbooks/disabling-network-connections-if-needed.html.md.erb

```diff
@@ -0,0 +1,106 @@
+---
+owner_slack: "#modernisation-platform"
+title: Disabling network access if required
+last_reviewed_on: 2024-02-14
+review_in: 6 months or when is resolved.
+---
+
+# <%= current_page.data.title %>
+
+## Overview
+
+There may be occassions where there is a network attack on modernisation platform environments that could require a way of stopping access. This document lists teo possible solutions to the issue.
+
+[NOTE]
+
+> All the steps below assume you can access the system through the AWS console and do not require changes to the terraform code.
```

Comment:
I think we would want to explore all ways of blocking access, so not just through the console, but also through code (which in some cases would be quicker than through the console)

Reply:
I did notice on place where that may be quicker and made a note about it. I have not, however, included the details and location on what would need to change. I will look into this to make sure it can be included.